x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/output_form_validators.tsx

```diff
@@ -369,6 +369,34 @@ export function validateDynamicKafkaTopics(value: Array<EuiComboBoxOptionOption<
         })
       );
     }
+
+    const getAllIndices = (str: string, substring: string): number[] => {
+      const indices = [];
+      let indx = str.indexOf(substring);
+      while (indx !== -1) {
+        indices.push(indx);
+        indx = str.indexOf(substring, indx + 1);
+      }
+      return indices;
+    };
+    // need to make sure that there are matching amounts of opening and closing brackets
+    const openingBracks = getAllIndices(val?.value!, '{[');
```

Comment:
I'm a little concerned about having to use `!` here. Could we look at getting an early return somewhere so we can guarantee we're not operating on falsey values here?